src/seagap/__init__.py

```diff
@@ -1,7 +1,9 @@
 from pkg_resources import DistributionNotFound, get_distribution
 
+package_name = __name__
```

Comment:
just curious, is this just to make things clearer?

Reply:
Yepp, it is 😄